jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "CircleCI API v2 introduction"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI Application Programming Interface (API) version 2, referred to as API v2 throughout this page, is a significant update to the existing CircleCI API v1.1 that is currently being used by internal and external audiences.
+
+* TOC 
+{:toc}
+
+## Introduction to API v2
+
+CircleCI API v2 has significantly changed the current API v1.1 paradigm to incorporate some important changes and shifts in the way that you can and will use the API for your jobs. Although changes were made to the available endpoints (discussed later on this page), two of the most important changes are the introduction of "pipelines" and "project slugs" as concepts in the API design and architecture, as well as a set of new endpoints that enable you to:
+
+- Retrieve a workflow by ID
+- Trigger pipelines with parameters
+- Trigger particular workflows within pipelines
+- Retrieve a pipeline by ID
+- Retrieve recent pipelines for an organization or for a project (ideally with branch filtering)
+
+**Note** In API v1.1, the term "build processing" was used to describe how projects were built and processed; however, with the API v2 release, "build processing" has been replaced by "pipelines." Not only has the terminology changed, but all projects have been moved over to this new setting by default.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated in much the same way as they would be authenticated in the previous API v1.1. A simple way to authenticate is to send your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the examnple shown below:
+
+`curl -u ${CIRCLECI_TOKEN}: https://circleci.com/api/v2/me`
```

Comment:
That `:` is optional, and I think it's more intuitive to omit it.

Reply:
~\~ It'd be great to use the long-form arg (`--user`) too, the long-forms can be much clearer to folks who aren't familiar with a CLI tool.